kamilkisiela/graphql-hive
# Improvements in change detection and the strictness of the registry

- Make `Query.latestVersion` nullable
- Make `Query.latestValidVersion` nullable
- Enforce service name in schema:check
- Remove `Built a new rate-limit map` log
- Pass SDL when available but orchestrator collected composition errors - to still detect changes between versions

### Background

Migrates some of the improvements from #789
